bears/kotlin/KotlinLintBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='ktlint',
+        global_bear=True,
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):'
+                     r'(?P<column>\d+): (?P<message>.+)')
+class KotlinLintBear:
+    """
+    Lints your Kotlin Files.
+    Check code for coding standards or semantical problems that might lead
+    to problems in execution of the code.
+    See https://ktlint.github.io for more info.
```

Comment:
This link can be added to the `SEE_MORE` metadata attribute.

Reply:
👍